estate/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import estate_property
+from . import estate_type
+from . import estate_tag
+from . import estate_offer
+from . import users
```

Comment:
```suggestion
from . import res_users
```
Model name is `res.users`, so here the file name should be res_users.py